CMakeLists.txt

```diff
@@ -8,65 +8,74 @@
 #  directory.
 #
 
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.10.2)
 
 project(Fenix C)
 # The version number.
 set(FENIX_VERSION_MAJOR 1)
 set(FENIX_VERSION_MINOR 0)
 
-option(BUILD_EXAMPLES  "Builds example programs from the examples directory"   OFF)
-option(BUILD_TESTING   "Builds tests and test modes of files"                  ON)
+option(FENIX_EXAMPLES    "Builds example programs from the examples directory" OFF)
+option(FENIX_TESTS       "Builds tests and test modes of files"                OFF)
```

Comment:
FWIW CTest automatically creates the `BUILD_TESTING` variable https://cmake.org/cmake/help/latest/module/CTest.html

It's probably better to use that since it's a kind of a convention. Though I think it's not very well known...

Reply:
I didn't know that, I've reverted these thanks!